ci/ci.js

```
@@ -207,22 +207,22 @@ module.exports = class CI {
      * Returns the JSON output of npm audit in the current path.
      */
     npmAudit() {
-        const tmpFile = '_audit.json';
-        let jsonFile = fs.openSync(tmpFile, 'w+');
-    
         console.log("npm audit --json");
         try {
-            e.execSync("npm audit --json", { 'stdio': ['pipe', jsonFile, jsonFile] });
-        } catch (e) {}
-    
-        let output = fs.readFileSync(tmpFile, { 'encoding': 'utf8' });
-    
-        // This potentially fails for an invalid audit result
-        output = JSON.parse(output);
-    
-        fs.unlinkSync(tmpFile);
-    
-        return output;
+            const tmpFile = '_audit.json';
+            let jsonFile = fs.openSync(tmpFile, 'w+');
+            try {
+                e.execSync("npm audit --json", { 'stdio': ['pipe', jsonFile, jsonFile] });
+            } catch (e) {}
+            let output = fs.readFileSync(tmpFile, { 'encoding': 'utf8' });
+            // This potentially fails for an invalid audit result
+            output = JSON.parse(output);
+            fs.unlinkSync(tmpFile);  
+            return output;
+        } catch (e) {
+            console.warn("npm audit failed");
```

@cjelger isn't the exception swallowed at line 216 that you want to catch here? Do you still need the inner try-catch block?

---

The fix for the NPM audit is a temporary fix, see CIF-694.